src/components/Reports/SubscriberHistory/SubscriberHistoryCurrentSearch/SubscriberHistoryCurrentSearch.js

```diff
@@ -80,22 +84,17 @@ const SubscriberHistoryCurrentSearch = ({ searchText, dependencies: { dopplerApi
                   <td>{subscriber.firstName ? subscriber.firstName.value : ''}</td>
                   <td>{subscriber.lastName ? subscriber.lastName.value : ''}</td>
                   <td>
-                    {subscriber.score}
-                    {/*missing the logic to take the ranking
-                    <div class="dp-calification">
-                        <span
-                          class="ms-icon icon-star"
-                          aria-label="Una estrella"
-                        ></span
-                        ><span
-                          class="ms-icon icon-star"
-                          aria-label="Sin estrella"
-                        ></span>
-                        <span
-                          class="ms-icon icon-star"
-                          aria-label="Sin estrella"
-                        ></span>
-                      </div> */}
+                    <div className="dp-calification">
```

Comment:
Another possibility would be instead of refactoring a function into utils, add a new component for calification, with the function and jsx alltogether. What do you think?

Reply:
i added new component